sdk/eventhub/event-hubs/samples/browserSample/README.md

```diff
@@ -0,0 +1,81 @@
+---
+page_type: sample
+languages:
+  - javascript
+products:
+  - azure
+  - azure-event-hubs
+urlFragment: event-hubs-javascript
+---
+
+# Azure Event Hubs client library samples for JavaScript
```

Comment:
This h1 is what shows up as the card title on docs.microsoft.com/samples :

![image](https://user-images.githubusercontent.com/666532/74060129-273a1600-499e-11ea-96e0-abfee8f217ff.png)

We don't want it to have the same title as the other set of JS samples, so you should change it to something like "Azure Event Hubs Browser Samples for JavaScript".

Reply:
Oh cool. To be honest I didn't realize this controlled how the sample displayed on docs.microsoft.com. Good to know and I'll make an update!

Reply:
Updated!

Reply:
There's a thing you can put in the YAML frontmatter that will set it, but the default if that key isn't set is to pull the first H1 on the page and use that.